docs/spec/wrapper.md

```diff
@@ -0,0 +1,19 @@
+# Sanitizations for client
+After generating the client using open api specification, the following modifications are made to the generated client by introducing a wrapper client.
+
+Remove the `string accountSid` parameter from all non-account-related functions and add it as an optional parameter to each function, with the default parameter set to `accountSid` in the initial client configurations.
+
+For example, the function:
+
+```ballerina
+remote isolated function createCall(string accountSid, CreateCallRequest payload) returns Call|error {
+}
+```
+
+should be restructured to:
```

Comment:
```suggestion
is restructured as:
```

Reply:
Addressed in [e53338e](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/e53338ef5bd6bd3cc6640678b571a81183fbdbbf)